src/irmin/s.ml

```diff
@@ -1119,11 +1119,15 @@ module type SYNC_STORE = sig
 
   type status = [ `Empty | `Head of commit ]
 
+  val status_t : db -> status Type.t
```

Comment:
I've now changed this to expose `status_t` instead. It requires parameterising on `db`, since `S.commit_t` needs an `S.t`. 

I'm not sure if it's worth exposing the intermediate type `commit_t` here; might make sense since this type is being kept abstract within `Sync`.

Reply:
Maybe that extra parameter has a too high cost for the end-user. What do you think? Both options are good for me (exposing `pp_status` or `status_t`), pick the one you prefer :-) Or maybe expose both functions?

Reply:
Yep, I think I agree. It doesn't feel right if the intention is just to make this value printable. I'll put the `pp_status` back, and we can table a more considered unification of the API types at a later date.

Reply:
Done :)